Automattic/jetpack
# VideoPress: add edit button to VideoDetailsCard component

This PR implements the edit button to the VideoDetailsCard component.

Fixes #

#### Changes proposed in this Pull Request:

* VideoPress: add edit button to VideoDetailsCard component. Improve story.


#### Other information:


#### Jetpack product discussion



#### Does this pull request change what data or activity we track or use?



#### Testing instructions:






* Use storybook
* Edit the props. Confirm the changes are propagated to the story
* Test how the component triggers the callback when clicking on the dropdown menu items


https://user-images.githubusercontent.com/77539/186491002-672a589d-64d4-49c3-9947-b22e5243e552.mov
